src/api/app/components/notification_action_description_component.rb

```diff
@@ -15,9 +16,20 @@ def call
       when 'Event::CommentForPackage'
         commentable = @notification.notifiable.commentable
         "#{commentable.project.name} / #{commentable.name}"
+      when 'Event::RelationshipCreate'
+        role = @notification.event_payload['role']
+        user = @notification.event_payload['who']
+        object = @notification.event_payload['package'] || @notification.event_payload['project']
+        "#{user} made you #{role} of #{object}"
+      when 'Event::RelationshipDelete'
+        role = @notification.event_payload['role']
+        user = @notification.event_payload['who']
+        object = @notification.event_payload['package'] || @notification.event_payload['project']
+        "#{user} removed you as #{role} of #{object}"
```

Comment:
Is the project name mentioned elsewhere? Most packages have multiple instances and context matters.

Reply:
Mm right, I'll fix that

Reply:
Fixed